metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -94,12 +96,69 @@ object ServerCommands {
        |""".stripMargin
   )
 
+  val GotoLocation = Command(
+    "goto",
+    "Goto location",
+    """|Move the cursor to the definition of the argument symbol.
+       |
+       |Arguments: [string], where the string is a SemanticDB symbol.
+       |""".stripMargin
+  )
+
   /**
    * Open the browser at the given url.
    */
   val OpenBrowser: Regex = "browser-open-url:(.*)".r
   def OpenBrowser(url: String): String = s"browser-open-url:$url"
 
+  val GotoLog = Command(
+    "goto-log",
+    "Check logs",
+    "Open the Metals logs to troubleshoot issues."
+  )
+
+  val OpenIssue = Command(
+    OpenBrowser("https://github.com/scalameta/metals/issues/new/choose"),
+    "Open issue on GitHub",
+    "Open the Metals repository on GitHub to ask a question, report a bug or request a new feature."
+  )
+
+  val StarMetals = Command(
```

Comment:
Starring is (hopefully ;) ) a one-time action. Also, calling it that way seems both manipulative and misleading (not to mention desperate ;P ). 

`See Metals/Bloop on GitHub` is what people would want to do much more often than starring it. And if someone will want to start it then hopefully he would also want to take a look around some issues, pull requests and feature requests to know what to expect in the future.

Reply:
I think it's fine, I don't think that is in any way manipulative and misleading. From my point of view, I also don't consider this to be a desperate action; starring a project is the least way of recognizing the time and effort put by the developers into creating software *for free*. I think it's fair to ask for it.

Reply:
Oh, I agree on the whole recognition through starring point - but that is not what I am picking on here.

I am just articulating my thoughts on the *name of the command* which is *constantly shown to the user* (even to one who have already starred it). 
In no way I am condemning the starring of a project. I am not saying "never ever ask for starts". I say - "here, in this place, it may not be always a relevant name". Nothing more, nothing less. 

If you think it is a good place to do so, then that is fine by me. I just believe that "See Metals on github" is relevant any time, while "Start Metals on github" is relevant only until you have actually starred it.

Reply:
> If you think it is a good place to do so, then that is fine by me. I just believe that "See Metals on github" is relevant any time, while "Start Metals on github" is relevant only until you have actually starred it.

Fair point, but putting "Star Metals on GitHub" as an action that people read will actually force them to realize "Hey, I use this a lot, maybe I should star it", whereas putting "See Metals on GitHub" will definitely not provoke that reaction.
  
I think letting people know there's a clear, visible way of giving recognition to a project they enjoy is benefitial for maintainers and help create a virtous cycle and attract more users through the GitHub repository.

Reply:
>I think letting people know there's a clear, visible way of giving recognition

OK, if that is the intention behind this action, then it is more than OK :)